rbac/management/group/view.py

```diff
@@ -605,6 +610,16 @@ def filters_from_params(self, valid_filters, model_name, request):
                 filters[f"{attr_filter_name}__icontains"] = param_value
         return filters
 
+    def validate_guid(self, uuid):
+        """Verify GUID provided is valid before attempting to retrieve it."""
+        try:
+            UUID(uuid)
+        except ValueError:
+            key = "groups uuid validation"
+            message = f"{uuid} is not a valid UUID."
+            raise serializers.ValidationError({key: _(message)})
+        return
```

Comment:
Nice! Overall LGTM! We could apply this to the /roles/ endpoint too! Nit: this return seems not necessary but not a big deal.

Reply:
Oh, yep, with you on the return. That was in there from stubbing things out. I'll pull that out real quick. After moving the function to utils.py as above, I can also pull it in to /roles/